### PR TITLE
Dev python day 2

### PR DESCRIPTION
**Pull request by Fblade25**
Benchmarks:
Part 1: 1394.5 us
Part 2: 1563.0 us

### CODE REVIEW

File: Python/challenges/day_2.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def get_result(round, strategy=False):
+    score_matrix = [3, 6, 0]
+    moves = round.split(" ")
+    opponent = int(ord(moves[0]) - 65)
+    you = int(ord(moves[1]) - 88)
```
Comment (Sizuni) [inline Python/challenges/day_2.py]: When working with `ord()`, subtracting by 65 may be confusing. Instead, consider replacing it with `ord('A')`.

File: Python/challenges/day_2.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    else:
+        predict_move_matrix = [0, 1, 2, 0, 1, 2]
+        you = predict_move_matrix[opponent - 1 + you]
+        return (score_matrix[3 - opponent:] + score_matrix[:3 - opponent])[you] + you + 1
+
+def a(input):
+    points = 0
+    for round in input:
+        points += get_result(round)
+    return points
```
Comment (Sizuni) [inline Python/challenges/day_2.py:17]: Nice and short main method

File: Python/challenges/day_2.py (hunk 1 above)
Comment (Fblade25) [reply Python/challenges/day_2.py]:
```suggestion
    opponent = int(ord(moves[0]) - ord("A"))
    you = int(ord(moves[1]) - ord("X"))
```
this does look at lot cleaner, good suggestion

File: Python/challenges/day_2.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    score_matrix = [3, 6, 0]
+    moves = round.split(" ")
+    opponent = int(ord(moves[0]) - ord("A"))
+    you = int(ord(moves[1]) - ord("X"))
+    if not strategy:
+        return (score_matrix[3 - opponent:] + score_matrix[:3 - opponent])[you] + you + 1
+    else:
+        predict_move_matrix = [0, 1, 2, 0, 1, 2]
+        you = predict_move_matrix[opponent - 1 + you]
+        return (score_matrix[3 - opponent:] + score_matrix[:3 - opponent])[you] + you + 1
```
Comment (Sizuni) [inline Python/challenges/day_2.py:11]: python array solution, gg

File: Python/challenges/day_2.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+def get_result(round, strategy=False):
```
Comment (Sizuni) [inline Python/challenges/day_2.py:1]: What if there's a third strategy?

**Review by Sizuni — APPROVED** [review]

File: Python/challenges/day_2.py (hunk 4 above)
Comment (Fblade25) [reply Python/challenges/day_2.py:1]:
```suggestion
def get_result(round, strategy=False, strategy2=False, ...):
```

Comment (Sizuni) [reply Python/challenges/day_2.py:1]: How will you handle someone calling `get_result("A X", True, True)`

Comment (Fblade25) [reply Python/challenges/day_2.py:1]: they will simply only have the output of the first strat as it returns you out of the function

**Pull request merged**